fixture/buildroot/glibc_generate_locale_fail.defconfig

```diff
@@ -0,0 +1,3 @@
+## This fixture should fail. BR2_GENERATE_LOCALE should be n if BR2_TOOLCHAIN_USES_GLIBC is y
```

Comment:
Might be some confusion here. This is actually the typical case where the system is using GLIBC. If they are, then they need the `en_US.UTF-8` locale to avoid the Elixir warning. (Part of me wants to see if `C.UTF-8` would suffice to avoid the US-centrism. See the RPi0. The reason that this passes is probably because `BR2_TOOLCHAIN_USES_GLIBC=y` is never in the `nerves_defconfig`. It's only in the `.config`. At least for now, the check should be against `BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y`.

Reply:
:+1: So instead of `BR2_TOOLCHAIN_USES_GLIBC` it should be `BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC`?

Reply:
Right.